policyengine_us/variables/household/income/person/is_eligible_for_fsla_overtime_protection.py

```diff
@@ -0,0 +1,56 @@
+from policyengine_us.model_api import *
+
+
+class is_eligible_for_fsla_overtime_protection(Variable):
+    value_type = bool
+    entity = Person
+    label = "is eligible for overtime pay"
+    reference = "https://www.law.cornell.edu/cfr/text/29/541.600 ; https://www.law.cornell.edu/uscode/text/29/213"
+    definition_period = YEAR
+
+    def formula_2014(person, period, parameters):
+        p = parameters(period).gov.irs.income.exemption.overtime
+        is_paid_hourly = person("is_paid_hourly", period)
+        employment_income = person("employment_income", period)
+        weekly_employment_income = employment_income / WEEKS_IN_YEAR
+
+        # Special case for computer science occupation due to different salary threshold
+        is_computer_exempt = (
+            person("is_computer_scientist", period)
+            & (not is_paid_hourly)
+            & (
+                weekly_employment_income / p.hours_threshold
+                >= p.computer_salary_threshold
+            )
+        )
+
+        # HCE exemption - highly compensated employees regardless of occupation
+        is_hce_exempt = (employment_income >= p.hce_salary_threshold) & (
+            not is_paid_hourly
```

Comment:
everywhere - needed for vectorization
```suggestion
            ~is_paid_hourly
```